scripts/test/run-integration-tests.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+OS_OVERRIDE=$(go env GOOS)
+pushd ./test/integration-new
+
+echo "Running integration test with the following configuration:
```

Comment:
Could we also expose an ENDPOINT env variable - https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/master/scripts/test/run-canary-test.sh#L21-L23